test/multiplier/txn.go

```diff
@@ -87,7 +87,7 @@ func (n *txnCommit) Apply(source action.Actions) action.Actions {
 			postCommitActions = action.Actions{}
 		}
 
-		if i > lastCreateStoreIndex && firstCloseIndex == 0 {
+		if i > lastCreateStoreIndex && i < firstCloseIndex {
```

Comment:
question: Haven't looked at the whole test, wondering very quickly why does this change?

Reply:
Sorry, I should have explained that.  `firstCloseIndex` is pretty much always non-zero at this point as I moved the setting of the var into a different loop, this meant this `if` block was never hit and the read/write actions for the multiplier were never wrapped by the transaction.